### PR TITLE
initial sponsors page

### PR DESCRIPTION
**Pull request by khjrtbrg**
Addresses issue https://github.com/OSFeels/OSFeels.com/issues/5

### CODE REVIEW

**Comment by brennx0r** [comment]
++++ To veggie use!
Also, COURGETTES!  SQUEE!

**Comment by strand** [comment]
Small columns of justified text can end up looking weird. Mind if I make this a two column layout with right-aligned text…

screen shot 2015-08-07 at 6 17 47 pm

Or justified text…

screen shot 2015-08-07 at 6 18 22 pm

???

**Comment by khjrtbrg** [comment]
@strand Either sounds good to me! :+1:

**Comment by strand** [comment]
To complete get descriptions for:

- [ ] GitHub
- [ ] HoneyBadger
- [ ] npm
- [ ] NIRD
- [ ] StickerMule
- [ ] Thomas Street

Get logos for:

- [ ] NIRD
- [ ] StickerMule

Nice to have:

- [ ] Extract the data to YML or JSON and iterate through the HTML in an each loop.

File: app/feels.rb [hunk 1]
```diff
@@ -12,6 +13,11 @@
   markdown :conduct, layout_engine: :erb
 end
 
+get '/sponsors' do
+  @sponsors = YAML.load File.read("app/data/sponsors.yml")
```
Comment (khjrtbrg) [inline app/feels.rb]: :tada: :+1:

**Comment by khjrtbrg** [comment]
lgtm :boat:

**Comment by strand** [comment]
Can we ship this and add other sponsors as details come in?

**Pull request merged**